VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/EstimatedDecisionDate/EstimatedDecisionDate.test.tsx

```diff
@@ -1,64 +1,42 @@
 import { DateTime } from 'luxon'
-
-import { context, mockNavProps, mockStore, render } from 'testUtils'
+import { context, mockNavProps, render } from 'testUtils'
 import { InitialState } from 'store/slices'
 import EstimatedDecisionDate from './EstimatedDecisionDate'
-import { AlertBox, TextView, VAButton } from 'components'
-import { RenderAPI } from '@testing-library/react-native'
 
 const mockExternalLinkSpy = jest.fn()
 
 jest.mock('utils/hooks', () => {
   const original = jest.requireActual('utils/hooks')
-  const theme = jest.requireActual('styles/themes/standardTheme').default
-
   return {
     ...original,
     useExternalLink: () => mockExternalLinkSpy,
   }
 })
 
 context('EstimatedDecisionDate', () => {
-  let store: any
-  let component: RenderAPI
-  let props: any
-  let testInstance: ReactTestInstance
-
   const initializeTestCase = (maxEstDate: string, showCovidMessage: boolean): void => {
-    props = mockNavProps({ maxEstDate, showCovidMessage })
-
-    component = render(<EstimatedDecisionDate {...props} />, { preloadedState: { ...InitialState } })
-    testInstance = component.UNSAFE_root
+    const props = mockNavProps({ maxEstDate, showCovidMessage })
+    render(<EstimatedDecisionDate {...props} />, { preloadedState: { ...InitialState } })
   }
 
-  beforeEach(() => {
-    initializeTestCase('2020-12-20', false)
-  })
-
   it('should initialize', async () => {
-    expect(component).toBeTruthy()
+    initializeTestCase('2020-12-20', false)
+    expect(screen.getByText('Estimated decision date:')).toBeTruthy()
+    expect(screen.getByText('December 20, 2020')).toBeTruthy()
+    expect(screen.getByText('We estimated your claim would be completed by now but we need more time.')).toBeTruthy()
   })
 
   describe('when showCovidMessage is true', () => {
-    beforeEach(async () => {
+    it('should show an AlertBox and VAButton and should launch external link when button is pressed', async () => {
       initializeTestCase('2020-12-20', true)
-    })
-
-    it('should show an AlertBox and VAButton', async () => {
-      expect(testInstance.findAllByType(AlertBox).length).toEqual(1)
-      expect(testInstance.findAllByType(VAButton).length).toEqual(1)
-    })
-
-    describe('on click of the va button', () => {
-      it('should launch external link', () => {
-        testInstance.findByType(VAButton).props.onPress()
-        expect(mockExternalLinkSpy).toHaveBeenCalled()
-      })
+      expect(screen.getByText("We can’t provide an estimated date on when your claim will be complete due to the affect that COVID-19 has had on scheduling in-person claim exams. We’re starting to schedule in-person exams again in many locations. To see the status of claim exams in your area, you can review locations where we’re now offering in-person exams.")).toBeTruthy()
+      expect(screen.getByText('Review locations')).toBeTruthy()
+      fireEvent.press(screen.getByText('Review locations'))
```

Comment:
Another place where `getByRole` can be used
```suggestion
      fireEvent.press(screen.getByRole('button', { name: 'Review locations' }))
```

Reply:
updated